lib/tokenize.cpp

```diff
@@ -5957,8 +5957,28 @@ void Tokenizer::dump(std::ostream &out) const
         // could result in invalid XML, so run it through toxml().
         outs += "str=\"";
         outs += ErrorLogger::toxml(dir.str);
+        outs +="\">";
+        outs += '\n';
+        for (const simplecpp::Token * tok = dir.startToken; tok && tok != dir.endToken; tok = tok->next) {
+            outs += "        <token ";
+            outs += "column=\"";
+            outs += std::to_string(tok->location.col);
+            outs += "\" ";
+            outs += "str=\"";
+            outs += ErrorLogger::toxml(tok->str());
+            outs +="\"/>";
+            outs += '\n';
+        }
+        outs += "        <token ";
+        outs += "column=\"";
+        outs += std::to_string(dir.endToken->location.col);
```

Comment:
in theory if endToken is nullptr then it crashes here.

Reply:
It's no theory. It crashes. ;) I'm fixing it